wts/blueprints/aggregate.py

```diff
@@ -0,0 +1,115 @@
+import asyncio
+import flask
+import httpx
+import time
+from cdiserrors import NotFoundError, UserError
+
+from ..auth import authenticate
+from ..models import db, RefreshToken
+from ..tokens import async_get_access_token
+
+
+blueprint = flask.Blueprint("aggregate", __name__)
+
+
+@blueprint.route("/<path:endpoint>", methods=["GET"])
+async def get_aggregate_response(endpoint):
+    """
+    Proxy GET requests to `endpoint` on each linked commons and return
+    an aggregated response.
+
+    For an authenticated request, each proxied request incudes an access token
+    fetched using the current user's refresh token.
+
+    The size of the aggregated response body can be reduced by supplying a
+    `filters` parameter. If provided, only return JSON key-value pairs whose
+    key is in `filters`. Multiple filters can be specified:
+
+    `GET /aggregate/user/user?filters=authz&filters=username`
+    """
```

Comment:
can we add the parameter definitions and return to the docstring here, not just the description